client/src/components/Navbar.jsx

```diff
@@ -27,20 +31,56 @@ const Navbar = () => {
               Trainers
             </Link>
           </li>
-          <li>
-            <button className="text-xl text-white font-poppins bg-gradient-to-r from-blue-500 to-blue-700 overflow-hidden hover:from-blue-600 hover:to-blue-800 py-3 px-6 rounded-full transform transition-all duration-500 ease-in-out hover:brightness-110 hover:animate-pulse">
-              <Link to="/login" className="font-poppins text-white text-xl">
-                Login
-              </Link>
-            </button>
-          </li>
-          <li>
-            <button className="text-xl text-white font-poppins bg-gradient-to-r from-blue-500 to-blue-700 overflow-hidden hover:from-blue-600 hover:to-blue-800 py-3 px-6 rounded-full transform transition-all duration-500 ease-in-out hover:brightness-110 hover:animate-pulse">
-              <Link to="/signup" className="font-poppins text-white text-xl">
-                Sign up
-              </Link>
-            </button>
-          </li>
+          {user != null ? (
+            <li>
+              <Dropdown
+                arrowIcon={false}
+                inline
+                label={
+                  <AccountCircleIcon
+                    fontSize="large"
+                    color="disabled"
+                    style={{ color: "white" }}
+                  />
+                }
+              >
+                <Dropdown.Header>
+                  <span className="font-poppins block text-sm">
+                    {user.name}
+                  </span>
+                  <span className="font-poppins block truncate text-sm font-medium">
+                    {user.email}
+                  </span>
+                </Dropdown.Header>
+                <Dropdown.Item className="font-poppins">
+                  Dashboard
+                </Dropdown.Item>
+                <Dropdown.Item className="font-poppins">Settings</Dropdown.Item>
+                <Dropdown.Divider />
+                <Dropdown.Item className="font-poppins">Sign Out</Dropdown.Item>
```

Comment:
- Wrap each `<Dropdown.Item>` with a `<Link>` and provide a prefix of `auth` as these are protected routes.
- Leave as 404 pages for now
- Additionally, on `Sign out` dispatch a logout from reducers-> reroute to landingPage.
   - reducer is setup already. You can pass `onClick` prop on Link
```suggestion
               <Link to={'/auth/dashboard'}>
                    <Dropdown.Item className="font-poppins">
                      Dashboard
                    </Dropdown.Item>
                </Link> 
                // ADD SETTINGS HERE
                 <Link to={'/'} onClick = {dispatch(logout())}>
                    <Dropdown.Item className="font-poppins">
                      Sign out
                    </Dropdown.Item>
                </Link> 
```